<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ruzik</author>
 <class>RizekQssEditorClass</class>
 <widget class="QMainWindow" name="RizekQssEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rizek QSS Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rizekqsseditor.qrc">
    <normaloff>:/ico/icos/package_editors.png</normaloff>:/ico/icos/package_editors.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="qssTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="documentTitle">
       <string>Rizek QSS Editor</string>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_projects">
     <property name="statusTip">
      <string>Your last projects</string>
     </property>
     <property name="title">
      <string>Recent projects</string>
     </property>
     <property name="icon">
      <iconset resource="rizekqsseditor.qrc">
       <normaloff>:/ico/icos/recentdocuments.png</normaloff>:/ico/icos/recentdocuments.png</iconset>
     </property>
    </widget>
    <widget class="QMenu" name="menuCurrent_file">
     <property name="title">
      <string>Current file</string>
     </property>
     <property name="icon">
      <iconset resource="rizekqsseditor.qrc">
       <normaloff>:/ico/icos/current file.png</normaloff>:/ico/icos/current file.png</iconset>
     </property>
     <addaction name="actionReopen"/>
     <addaction name="actionOpen_file_in_standart_note"/>
     <addaction name="actionOpen_dir_of_current_file"/>
    </widget>
    <widget class="QMenu" name="menuView">
     <property name="title">
      <string>View</string>
     </property>
     <property name="icon">
      <iconset resource="rizekqsseditor.qrc">
       <normaloff>:/ico/icos/view.png</normaloff>:/ico/icos/view.png</iconset>
     </property>
     <addaction name="actionStandart"/>
     <addaction name="actionLeft_bottom"/>
     <addaction name="actionRight_bottom"/>
     <addaction name="actionClear"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_projects"/>
    <addaction name="menuCurrent_file"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="menuView"/>
    <addaction name="separator"/>
    <addaction name="actionHide_to_tray"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionReplace"/>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_all"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuQSS_Documentation">
     <property name="title">
      <string>QSS Documentation</string>
     </property>
     <property name="icon">
      <iconset resource="rizekqsseditor.qrc">
       <normaloff>:/ico/icos/textdocument.png</normaloff>:/ico/icos/textdocument.png</iconset>
     </property>
     <addaction name="actionDocumentation"/>
     <addaction name="actionExamples"/>
    </widget>
    <addaction name="menuQSS_Documentation"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionDonate"/>
    <addaction name="actionCheck_for_updates"/>
    <addaction name="actionAbout_program"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuInsert">
     <property name="title">
      <string>Insert special item</string>
     </property>
     <property name="icon">
      <iconset resource="rizekqsseditor.qrc">
       <normaloff>:/ico/icos/insert.png</normaloff>:/ico/icos/insert.png</iconset>
     </property>
     <addaction name="actionColor"/>
     <addaction name="actionFont_name"/>
    </widget>
    <addaction name="actionPreview"/>
    <addaction name="actionSearch_mistakes"/>
    <addaction name="actionResources"/>
    <addaction name="separator"/>
    <addaction name="menuInsert"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionOutput"/>
    <addaction name="actionScript"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/open.png</normaloff>:/ico/icos/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open file with your qss style sheet(qss files has extension *.qss)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/save.png</normaloff>:/ico/icos/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save your style sheet in current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/save as.png</normaloff>:/ico/icos/save as.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="statusTip">
    <string>Save your style sheet in file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/replace.png</normaloff>:/ico/icos/replace.png</iconset>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="statusTip">
    <string>Replace some words in your style sheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/Documentation.png</normaloff>:/ico/icos/Documentation.png</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="statusTip">
    <string>Look QSS documentation on the Qt website</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionExamples">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/examples.png</normaloff>:/ico/icos/examples.png</iconset>
   </property>
   <property name="text">
    <string>Examples</string>
   </property>
   <property name="statusTip">
    <string>Look QSS examples on the Qt website</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionAbout_program">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/home.png</normaloff>:/ico/icos/home.png</iconset>
   </property>
   <property name="text">
    <string>About program</string>
   </property>
   <property name="statusTip">
    <string>Look some information about Rizek QSS Editor</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/find.png</normaloff>:/ico/icos/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="statusTip">
    <string>Find text in the document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/view.png</normaloff>:/ico/icos/view.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Create new file of Qt style sheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/document-preview.png</normaloff>:/ico/icos/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="statusTip">
    <string>Preview your design in a separate window</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/settings.png</normaloff>:/ico/icos/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="statusTip">
    <string>Change settings of your program</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/print.png</normaloff>:/ico/icos/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="statusTip">
    <string>Print text of your current project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/colors.png</normaloff>:/ico/icos/colors.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="statusTip">
    <string>Insert color(#xxxxxx) in your code</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionFont_name">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/font.png</normaloff>:/ico/icos/font.png</iconset>
   </property>
   <property name="text">
    <string>Font name</string>
   </property>
   <property name="statusTip">
    <string>Insert font name in your code</string>
   </property>
   <property name="shortcut">
    <string>Alt+F</string>
   </property>
  </action>
  <action name="actionReopen">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/open.png</normaloff>:/ico/icos/open.png</iconset>
   </property>
   <property name="text">
    <string>Reopen</string>
   </property>
  </action>
  <action name="actionOpen_file_in_standart_note">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/open.png</normaloff>:/ico/icos/open.png</iconset>
   </property>
   <property name="text">
    <string>Open the file through the system</string>
   </property>
  </action>
  <action name="actionSearch_mistakes">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/error.png</normaloff>:/ico/icos/error.png</iconset>
   </property>
   <property name="text">
    <string>Search syntax mistakes(beta)</string>
   </property>
   <property name="statusTip">
    <string>Search syntax mistakes in your project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionOutput">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/Output.png</normaloff>:/ico/icos/Output.png</iconset>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
   <property name="statusTip">
    <string>Output messages from program</string>
   </property>
  </action>
  <action name="actionStandart">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/View_Standart.png</normaloff>:/ico/icos/View_Standart.png</iconset>
   </property>
   <property name="text">
    <string>Standart</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionLeft_bottom">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/View_Left-Bottom.png</normaloff>:/ico/icos/View_Left-Bottom.png</iconset>
   </property>
   <property name="text">
    <string>Left-bottom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionRight_bottom">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/View_Right-Bottom.png</normaloff>:/ico/icos/View_Right-Bottom.png</iconset>
   </property>
   <property name="text">
    <string>Right-bottom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/View_Clear.png</normaloff>:/ico/icos/View_Clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionOpen_dir_of_current_file">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/open.png</normaloff>:/ico/icos/open.png</iconset>
   </property>
   <property name="text">
    <string>Open dir of current file</string>
   </property>
  </action>
  <action name="actionHide_to_tray">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/hideToTray.png</normaloff>:/ico/icos/hideToTray.png</iconset>
   </property>
   <property name="text">
    <string>Hide to tray</string>
   </property>
   <property name="statusTip">
    <string>Hide your prgram in system tray</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/quit.png</normaloff>:/ico/icos/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string extracomment="Close Rizek Qss Editor"/>
   </property>
  </action>
  <action name="actionDonate">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/donate.png</normaloff>:/ico/icos/donate.png</iconset>
   </property>
   <property name="text">
    <string>Donate</string>
   </property>
   <property name="statusTip">
    <string>Do you want to speed up the release of a new version Rizek Qss Editor or our other programs?</string>
   </property>
  </action>
  <action name="actionResources">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/Resources.png</normaloff>:/ico/icos/Resources.png</iconset>
   </property>
   <property name="text">
    <string>Resources</string>
   </property>
   <property name="statusTip">
    <string>Open a window to work with resources</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/update.png</normaloff>:/ico/icos/update.png</iconset>
   </property>
   <property name="text">
    <string>Check for updates</string>
   </property>
   <property name="statusTip">
    <string>Chek Rizek Qss Editor for new version</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/Cut.png</normaloff>:/ico/icos/Cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/Copy.png</normaloff>:/ico/icos/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/insert.png</normaloff>:/ico/icos/insert.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/remove.png</normaloff>:/ico/icos/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/select_all.png</normaloff>:/ico/icos/select_all.png</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/Undo.png</normaloff>:/ico/icos/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/Redo.png</normaloff>:/ico/icos/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionScript">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/script.png</normaloff>:/ico/icos/script.png</iconset>
   </property>
   <property name="text">
    <string>Script</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="rizekqsseditor.qrc">
     <normaloff>:/ico/icos/Help.png</normaloff>:/ico/icos/Help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qssTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qssTextEdit/qssTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rizekqsseditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_all</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
